functional-tests/envs/testenv.py

```diff
@@ -113,27 +113,29 @@ def withdraw(
         self,
         ctx: flexitest.RunContext,
         el_address: str,
-        withdraw_address: str,
+        destination: str,
     ):
         """
-        Perform a withdrawal from the L2 to the given BTC withdraw address.
+        Perform a withdrawal from the L2 to the given BTC withdraw destination.
         Returns (l2_tx_hash, tx_receipt, total_gas_used).
+
+        NOTE: The withdrawal destination is a Bitcoin Output Script Descriptor (BOSD).
         """
         cfg: RollupConfig = ctx.env.rollup_cfg()
         # D BTC
         deposit_amount = cfg.deposit_amount
-        # Build the p2tr pubkey from the withdraw address
-        change_address_pk = extract_p2tr_pubkey(withdraw_address)
-        self.debug(f"Change Address PK: {change_address_pk}")
+        # Assert is a valid BOSD
+        assert is_valid_bosd(destination), "Invalid BOSD"
+        self.debug(f"Withdrawal Destination: {destination}")
```

Comment:
I thought we were using `self.info` instead of `self.debug`.

Reply:
Maybe, I need to rebase don't know when that directive went into effect.

Reply:
Still `testenv.py` is filled with `self.debug`. To be honest, `self.info` are really important for maybe CI-hosted functional tests.
Resolving this but we can move this to a discussion in Slack if needed.